addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/scanrules/ScriptsPassiveScanner.java

```diff
@@ -50,11 +50,28 @@ public ScriptsPassiveScanner() {
                                 Configuration.<PassiveScript>builder()
                                         .setScriptType(ExtensionPassiveScan.SCRIPT_TYPE_PASSIVE)
                                         .setTargetInterface(PassiveScript.class)
-                                        .setInterfaceErrorMessageProvider(
-                                                sw ->
-                                                        Constant.messages.getString(
-                                                                "scripts.scanRules.pscan.interfaceError",
-                                                                sw.getName()))
+                                        .setInterfaceProvider(
+                                                (scriptWrapper, targetInterface) -> {
+                                                    if (extension.getInterface(
```

Comment:
This does not seem to be working as expected, I added a Groovy script (without metadata) and it still returned non-null here.

Reply:
Yeah, the behaviour is different for some engines. Will take a look.